esf_main.tex

```diff
@@ -300,11 +307,19 @@ \subsubsection{Description/additional circuitry}
 
 \subsubsection{Wiring}
 %Describe the wiring, show schematics including the circuit board, show data regarding the cables and connectors used.  If not detailed in section 2.1, be sure to show how the device open the shutdown circuit.
-TODO
+
+TODO: Which board is this connected to with the relay on it? 
```

Comment:
This board will have a relay on it, as far as I'm aware. The shutdown circuit already runs up to the bulkhead anyway, so it's not a problem to have this board interrupt it.